inductiva/_cli/cmd_tasks/kill.py

```diff
@@ -0,0 +1,22 @@
+"""Kills a tasks by id via CLI."""
+
+import inductiva
+
+
+def kill_task(args):
+    """Kills a task by id."""
+    inductiva.tasks.Task(args.id).kill(wait_timeout=args.wait_timeout)
+
+
+def register(parser):
+    """Register the kill task command."""
+
+    subparser = parser.add_parser("kill", help="Kill a running task.")
+    subparser.add_argument("id", type=str, help="Id of the task to kill.")
+    subparser.add_argument("-w",
+                           "--wait-timeout",
+                           type=float,
+                           default=None,
+                           help="Timeout to wait for the kill to complete.")
+
```

Comment:
The user should be able to confirm he want to kill all tasks:

```suggestion
subparser.add_argument("-y", "--yes", action='store_true', ...)
```

Reply:
Does the confirmation work for the multiple tasks or should be for each individual one?

Reply:
I've made it one confirmation for all.